samples/frameworks/react/ts/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "typescript-react",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@azure/event-hubs": "^5.3.1",
+    "@azure/identity": "^1.2.0",
+    "@azure/service-bus": "^7.0.0",
+    "@azure/storage-blob": "^12.3.0",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-scripts": "4.0.1",
+    "typescript": "^4.0.3",
+    "uuid": "^8.3.1"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
```

Comment:
Just curious, does this also compile the TypeScript for you?

Reply:
Yup! It compiles it all down to JS